WordPress/UITestsFoundation/Screens/MySiteScreen.swift

```diff
@@ -196,6 +217,20 @@ public class MySiteScreen: ScreenObject {
         return self
     }
 
+    @discardableResult
+    public func verifyPagesCard() -> Self {
+        XCTAssertTrue(pagesCardHeaderButton.waitForIsHittable(), "Pages card: header not displayed.")
+        XCTAssertTrue(pagesCard.buttons["More"].waitForIsHittable(), "Pages card: context menu not displayed.")
```

Comment:
Firstly, I wanted to reply with:
`This is because "More" button is not used anywhere else besides this assertion (e.g. not tapped). Creating a getter for it in this case would not be justified, IMO (however, it's a matter of preference, too).`

However, I then noticed that `Create Page` button is also used only in assertion once, but still has a getter.

To make it consistent, I created a getter for `More` button too in 865dd4194b6996bd167ef7cceef3c94d8c6ac269.